docs(service-resolver): clarify the default time unit in service-resolver.ConnectTimeout (#16149)
hashicorp/consul: website/content/docs/connect/config-entries/service-resolver.mdx

      name: 'ConnectTimeout',
      type: 'duration: 0s',
      description:

>         'The timeout for establishing new network connections to this service. If time unit isn't provided, the default unit is nanosecond',

Reviewer: ```suggestion
        'The timeout for establishing new network connections to this service. The default unit of time is `ns`.',
```
Per the style guide, we want to avoid using contractions. And labeling the behavior as "default" makes the statement about not providing a unit unnecessary. Also, it might be better to use the actual unit value instead of writing out nanosecond just because it's closer to the actual usage, but I defer to you on that point.
